packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -309,6 +310,12 @@ const ReactTestUtils = {
    * @return {object} the ReactTestUtils object (for chaining)
    */
   mockComponent: function(module, mockTagName) {
+    lowPriorityWarning(
```

Comment:
This needs to be behind a duplicate check. Otherwise it might significantly spam console in tests.

Reply:
👍 Sure